### PR TITLE
Fixes, New board setting, Antispam

### PR DESCRIPTION
**Pull request by ghost**
The commit messages should be mostly self-explanatory. This is part one of some work I want to do this month, but don't expect anything else for a while. Some extra details below
- **Ban action also deletes post**
Having to manually bin a post that you ban is annoying and it makes dealing with automated spam difficult, since you have to cycle the mod panel from ban to delete post and back again while spam continues.
- **Forced liveposting board setting**
This adds a board setting that ignores (but does not change) users "live" identity setting when enabled. This mitigates some forms of text spam, but I'm adding it mostly because it's funny.
- **Port "Delete captcha tokens on ban" from meguca**
This invalidates the captcha token associated with a specific IP when it's banned, so a ban evader will always have to solve a new captcha before posting.
- **Map start and stop of migrations function**
Personal preference because it's long as hell and being able to jump to the bottom makes it harder to screw up database migrations.

Fyi both the forced liveposting and captcha token commits will require database upgrades. This worked fine on my local master clone but will cause issues if the local db/migrations.go code has diverged. Also it would be nice if you pushed my purple text commits locally.

### CODE REVIEW

**Comment by nekto-nekto** [comment]
> Also it would be nice if you pushed my purple text commits locally.

Done. I didn't have time to do it last time, i'm sorry.

I'll accept this PR a little later.

File: clientScripts/loader.js [hunk 1]
```diff
@@ -177,7 +177,7 @@
 		new URLSearchParams({q: "+"}).get('q') === "+" || loadScript("js/scripts/polyfill-url-search-params");
 
 		// fast dirty hacks for users
-		loadScript("js/scripts/fasthacksforusers");
+		//loadScript("js/scripts/fasthacksforusers");
```
Comment (nekto-nekto) [inline clientScripts/loader.js]: why? 
```suggestion
		loadScript("js/scripts/fasthacksforusers");
```

**Comment by ghost** [comment]
The fasthacksforusers script triggered constant page refreshes for several minutes when loaded on a firefox profile that didn't already have the meguca IndexedDB. It didn't happen as much on chromium or a firefox install that already has the local DB.

I can't reproduce the issue on the live instance at all. Something similar did used to happen when loading the site with Tor browser but now it must be a local issue with the testing VM.

**Comment by nekto-nekto** [comment]
Thanks.

**Pull request merged**